### DISCUSSION TITLE
How to apply a class to the component wrapper?

### DISCUSSION THREAD

**Discussion by TheDirigible** [inkline/inkline#309] [Need help]
Example:
<i-number-input classes="small" />
The class "small" is applied to the inner input, but I need a class applied to the outer wrapper, so all parts of the component can react to "small"

**Comment by alexgrozav** [comment]
Hi @TheDirigible,

You could either use `<i-input class="someclass">` directly, or apply he size to an `<i-form-group>`. 

Let me know if this helps.

**Comment by TheDirigible** [comment]
As I said, that applies the class to the inner <input>, not the wrapper.
